cdap-common/src/main/java/co/cask/cdap/common/security/AuthEnforceRewriter.java

```diff
@@ -259,25 +282,66 @@ public void visitEnd() {
               entityPart = new EntityPartDetail(fieldName, true);
```

Comment:
The previous line, `fieldDetails.containsKey(fieldName)`, what's the purpose of call it without using the result?